src/functionals/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+target_sources(xcfun-objlib
+  PRIVATE
+    ${CMAKE_CURRENT_SOURCE_DIR}/aliases.cpp
```

Comment:
Private sources do not need full path (but it does not harm).

Reply:
The full path is needed because I am creating an `OBJECT` library, apparently. But the file is a bit more compact now.

Reply:
Aha. Good point.